rotkehlchen/utils/network.py

```diff
@@ -7,7 +7,7 @@
 import requests
 
 from rotkehlchen.constants import GLOBAL_REQUESTS_TIMEOUT
-from rotkehlchen.constants.timing import DEFAULT_TIMEOUT_TUPLE, QUERY_RETRY_TIMES
+from rotkehlchen.db import settings
```

Comment:
again with the incosistency

```suggestion
from rotkehlchen.db.settings import CachedSettings
```

Reply:
Same thing here. You ignored my comment.